### DISCUSSION TITLE
Reach Internet

### DISCUSSION THREAD

**Discussion by P1c1s** [Kidel/Kathara#317] [Q&A] [answered]
Hi, I am using Kathará for a networking course at Roma 3 University. I was wondering if it was possible to create a network scenario where i can to reach the internet, for example make a ping to Google (8.8.8.8).  
Thanks advanced.

**Comment by tcaiazzi** [comment, marked as answer]
Hi @P1c1s, 

Thanks for reaching out!

Yes, it is possible to connect a device to the host network using the `bridged` option. 

From the man pages:  
> *"This option connects the device to the host network by adding an additional network interface. This interface will be connected to the host network through a NAT connection."*

For example, if you have a device called `pc1`, you can modify the `lab.conf` file as follows:  
```plaintext
pc1[0]="A"
pc1[bridged]="true"
```

When you start `pc1`, an additional interface (in this case `eth1`) will appear. This interface connects to the host network, enabling the device to access the Internet. Additionally, the device will automatically be configured to add a default route through the host network.

**Note:** Ensure the `/etc/resolv.conf` file is configured correctly with a nameserver, if you want to use DNS resolution.

Please let me know if this helps! 

Thanks!

**Reply by P1c1s** [reply]
Thank you so much. It works!!! Now the pc1 is connected with the docker network and I can reach internet. Thanks.
